jest.config.js

```diff
@@ -9,6 +9,8 @@ module.exports = {
   setupTestFrameworkScriptFile: '<rootDir>/src/setupEnzyme.ts',
```

Comment:
Replace with `setupFilesAfterEnv: ['<rootDir>/src/setupEnzyme.ts'],` because this showing warning when running tests.